src/components/ResponsiveSidebar.js

```diff
@@ -0,0 +1,149 @@
+import {
+  Box,
+  Drawer,
+  Divider,
+  Typography,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Stack,
+} from '@mui/material';
+import {
+  HomeOutlined,
+  AccountBalanceWalletOutlined,
+  AccountCircleOutlined,
+  ContactSupportOutlined,
+  PowerSettingsNewOutlined,
+} from '@mui/icons-material';
+
+export const drawerWidth = 241;
+
+function ResponsiveSidebar() {
+  const drawer = (
```

Comment:
This is the content of the drawer, and you can extract this into a separate React component, in a separate file, for better readability. 
```suggestion
  const DrawerContent = () => {
     ...
  }
```